app/modules/user/stats.rb

```diff
@@ -299,6 +304,7 @@ def sales_data_for_balance_ids(balance_ids)
       taxes_cents: taxes_cents_for_balances(balance_ids),
       affiliate_credits_cents: affiliate_credit_cents_for_balances(balance_ids),
       affiliate_fees_cents: affiliate_fee_cents_for_balances(balance_ids),
+      carried_over_cents_from_failed_payout: carried_over_cents_from_failed_payout_for_balances(balance_ids),
```

Comment:
The `sales_cents`, `taxes_cents`, etc. above already include the carried over balance as well, so this will break the math by double counting that amount I think?